From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Fabian=20Gr=C3=BCnbichler?= <f.gruenbichler@proxmox.com>
Date: Wed, 22 May 2019 14:08:37 +0200
Subject: [PATCH kronosnet] update changelog
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Signed-off-by: Fabian Grünbichler <f.gruenbichler@proxmox.com>
---
 debian/changelog | 20 ++++++++++++++++++++
 1 file changed, 20 insertions(+)

diff --git a/debian/changelog b/debian/changelog
index 63d5a6d..4872847 100644
--- a/debian/changelog
+++ b/debian/changelog
@@ -1,3 +1,23 @@
+kronosnet (1.10-pve2) pve; urgency=medium
+
+  * cherry-pick pmtud fixes
+
+ -- Proxmox Support Team <support@proxmox.com>  Fri, 2 Aug 2019 10:53:46 +0200
+
+kronosnet (1.10-pve1) pve; urgency=medium
+
+  * update to 1.10
+
+ -- Proxmox Support Team <support@proxmox.com>  Wed, 19 Jun 2019 09:32:21 +0200
+
+kronosnet (1.8-pve1) pve; urgency=medium
+
+  * introduce kronosnet for PVE 6.x
+
+  * cherry-pick crypto patches for Corosync 2.x keyfile compat
+
+ -- Proxmox Support Team <support@proxmox.com>  Wed, 22 May 2019 14:08:15 +0200
+
 kronosnet (1.8-2) unstable; urgency=medium
 
   * [b6a2cdc] New patch: send test: skip the SCTP test if SCTP is not supported
