From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Fabian=20Gr=C3=BCnbichler?= <f.gruenbichler@proxmox.com>
Date: Wed, 19 Jun 2019 10:41:22 +0200
Subject: [PATCH kronosnet] add new symbols for libknet 1.10
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Signed-off-by: Fabian Grünbichler <f.gruenbichler@proxmox.com>
---
 debian/libknet1.symbols | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/debian/libknet1.symbols b/debian/libknet1.symbols
index 0fa1fd5..eb0baa3 100644
--- a/debian/libknet1.symbols
+++ b/debian/libknet1.symbols
@@ -11,6 +11,7 @@ libknet.so.1 libknet1 #MINVER#
  knet_handle_clear_stats@LIBKNET 0.9
  knet_handle_compress@LIBKNET 0.9
  knet_handle_crypto@LIBKNET 0.9
+ knet_handle_enable_access_lists@LIBKNET 1.10
  knet_handle_enable_filter@LIBKNET 0.9
  knet_handle_enable_pmtud_notify@LIBKNET 0.9
  knet_handle_enable_sock_notify@LIBKNET 0.9
@@ -37,6 +38,8 @@ libknet.so.1 libknet1 #MINVER#
  knet_host_remove@LIBKNET 0.9
  knet_host_set_name@LIBKNET 0.9
  knet_host_set_policy@LIBKNET 0.9
+ knet_link_add_acl@LIBKNET 1.10
+ knet_link_clear_acl@LIBKNET 1.10
  knet_link_clear_config@LIBKNET 0.9
  knet_link_get_config@LIBKNET 0.9
  knet_link_get_enable@LIBKNET 0.9
@@ -45,6 +48,8 @@ libknet.so.1 libknet1 #MINVER#
  knet_link_get_pong_count@LIBKNET 0.9
  knet_link_get_priority@LIBKNET 0.9
  knet_link_get_status@LIBKNET 0.9
+ knet_link_insert_acl@LIBKNET 1.10
+ knet_link_rm_acl@LIBKNET 1.10
  knet_link_set_config@LIBKNET 0.9
  knet_link_set_enable@LIBKNET 0.9
  knet_link_set_ping_timers@LIBKNET 0.9
