From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Thomas Lamprecht <t.lamprecht@proxmox.com>
Date: Thu, 23 May 2019 20:57:21 +0200
Subject: [PATCH] d/control: switch back to comapt level 10 for stretch

Signed-off-by: Thomas Lamprecht <t.lamprecht@proxmox.com>
---
 debian/compat  | 2 +-
 debian/control | 4 ++--
 2 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/debian/compat b/debian/compat
index f599e28..b4de394 100644
--- a/debian/compat
+++ b/debian/compat
@@ -1 +1 @@
-11
+10
diff --git a/debian/control b/debian/control
index d7152ef..918b93c 100644
--- a/debian/control
+++ b/debian/control
@@ -4,9 +4,9 @@ Priority: optional
 Maintainer: Debian HA Maintainers <debian-ha-maintainers@lists.alioth.debian.org>
 Uploaders:
  Ferenc Wágner <wferi@debian.org>,
-Standards-Version: 4.3.0
+Standards-Version: 4.1.3
 Build-Depends:
- debhelper (>= 11~),
+ debhelper (>= 10~),
  libnl-3-dev,
  libnl-route-3-dev,
  libsctp-dev [linux-any],
